src/modules/Transactions/components/Detail/index.tsx

```diff
@@ -57,13 +57,48 @@ class Index extends PureComponent<IndexProps> {
     const isInitialLoad = !transaction;
     const events = get(transaction, 'hits.hits[0]._source.events', []);
     const eventsTable: any[] = [];
-    events.forEach((event: any) => {
+
+    function toObject(data: {}):string {
+      return JSON.stringify(data, (key, value) => {
+        if (typeof value === 'bigint') {
+          return value.toString();
+        } else if (typeof value === 'object') {
+          return value;
+        }
+        return value;
+      });
+    }
```

Comment:
this toObject  function occurs at least  twice, then should be exported as a helper function, and be imported at different files.

Reply:
```
if (typeof value === 'bigint') {
  return value.toString();
} else if (typeof value === 'object') {
  return value;
}
return value;

```
should be

```
if (typeof value === 'bigint') {
  return value.toString();
} 
return value;
```